lang/cs/Org.Apache.REEF.IMRU/OnREEF/Driver/IMRUDriver.cs

```diff
@@ -311,7 +311,7 @@ private void AddGroupCommunicationOperators()
             try
             {
                 TangFactory.GetTang()
-                    .NewInjector(mapInputPipelineDataConverterConfig)
+                    .NewInjector(mapOutputPipelineDataConverterConfig)
                     .GetInstance<IPipelineDataConverter<TMapOutput>>();
```

Comment:
If wrong is given it will assign default one which means pipelining will
not happen.
On Sep 27, 2015 2:22 AM, "Julia Wang" <notifications@github.com> wrote:

> In lang/cs/Org.Apache.REEF.IMRU/OnREEF/Driver/IMRUDriver.cs
> <https://github.com/apache/incubator-reef/pull/526#discussion_r40502005>:
>
> > @@ -311,7 +311,7 @@ private void AddGroupCommunicationOperators()
> >              try
> >              {
> >                  TangFactory.GetTang()
> > -                    .NewInjector(mapInputPipelineDataConverterConfig)
> > +                    .NewInjector(mapOutputPipelineDataConverterConfig)
> >                      .GetInstance<IPipelineDataConverter<TMapOutput>>();
>
> Would it matter if a wrong configuration is given? When you put wrong one,
> it didn't impact any test case. Is it because test case doesn't cover this
> scenario or because a wrong configuration also works?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-reef/pull/526/files#r40502005>.
>